inference/chat_completion.py

```diff
@@ -57,6 +59,18 @@ def main(
     torch.cuda.manual_seed(seed)
     torch.manual_seed(seed)
     model = load_model(model_name, quantization)
+    if use_fast_kernels:
```

Comment:
For the PEFT model case, does this need to happen after the PEFT/Lora weights have been merged?

Reply:
sure, updated works both way.